test/test_sparse_csr.py

```diff
@@ -1979,10 +1979,6 @@ def maybe_transpose(cond, m):
                                       *[torch.bfloat16] if SM80OrLater else [],
                                       *[torch.half] if SM53OrLater else [],
                                       *[torch.complex128] if CUSPARSE_SPMM_COMPLEX128_SUPPORTED else []))
-    @skipCUDAIf(
-        not _check_cusparse_spgemm_available(),
-        "cuSparse Generic API SpGEMM is not available"
-    )
```

Comment:
This is it's definition of _check_cusparse_spgemm_available(),(https://github.com/pytorch/pytorch/blob/b7b44e766b4e0fd620379854f9db433efd534886/test/test_sparse_csr.py#L44) 
so, removing it will not affect the other functionalities. 

```
def _check_cusparse_spgemm_available():
    # cusparseSpGEMM was added in 11.0
    return not TEST_WITH_ROCM
```